File: app/src/main/java/com/shop440/Models/CategoryModel.java
package com.shop440.Models;

import java.io.Serializable;

/**
 * Created by SMILECS on 1/24/17.
 */

public class CategoryModel implements Serializable {
    public String Name, Slug;

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public String getSlug() {
        return Slug;
    }

    public void setSlug(String slug) {
        Slug = slug;
    }

    @Override
    public String toString() {
        return getName();
    }
}

File: app/src/main/java/com/shop440/models/CategoryModel.kt
package com.shop440.models

import com.google.gson.annotations.SerializedName
import io.realm.RealmObject
import java.io.Serializable

/**
 * Created by SMILECS on 1/24/17.
 */

open class CategoryModel : RealmObject(){
    @SerializedName("Name") var catName: String = ""
    @SerializedName("Slug") var slug: String = ""
}